octopoes/requirements.txt

```diff
@@ -2,8 +2,10 @@ amqp==5.1.1 ; python_version >= "3.8" and python_version < "4.0"
 anyio==3.7.0 ; python_version >= "3.8" and python_version < "4.0"
 asgiref==3.7.2 ; python_version >= "3.8" and python_version < "4.0"
 backoff==2.2.1 ; python_version >= "3.8" and python_version < "4.0"
-billiard==3.6.4.0 ; python_version >= "3.8" and python_version < "4.0"
-celery==5.2.7 ; python_version >= "3.8" and python_version < "4.0"
+backports-zoneinfo==0.2.1 ; python_version >= "3.8" and python_version < "3.9"
+backports-zoneinfo[tzdata]==0.2.1 ; python_version >= "3.8" and python_version < "3.9"
+billiard==4.1.0 ; python_version >= "3.8" and python_version < "4.0"
+celery==5.3.0 ; python_version >= "3.8" and python_version < "4.0"
```

Comment:
This upgrades a lot more than just opentelemetry...

Reply:
I used `make export-requirements` to update the requirements files. Because all dependencies are in the version ranges we specified in the `project.toml` I think these updates from `poetry update` are expected behaviour?

Reply:
`make export-requirements` only exports the dependencies to a `requirements.txt` format and should be run after any poetry calls.

`poetry update` will actually upgrade (as in update the versions and install the packages) and modify the environment (it will respect carets and tildes). After that, it will update the lock file.

So yes, the updates from `poetry update` are expected behavior, but probably not something you wanted to do. We usually wait on Dependabot for (security and major) updates and review/ test the updates before including them in our package requirements. This might be a bit cumbersome (although poetry does make it easier to upgrade multiple packages and write the lock file), but easier to manage and more secure given that we don't have great test coverage (yet).

Reply:
I've redone the changes to the Poetry lockfile using `poetry lock --no-update` and then ran `make export-requirements` to create the requirements files. This time, no other dependencies were updated.